Vagrantfile

```diff
@@ -100,6 +107,10 @@ Vagrant.configure("2") do |global_config|
       if vagrant_boxes.key? vagrant_box
         config.vm.box_url = vagrant_boxes[vagrant_box]
       end
+      if vagrant_box_aliases.key? vagrant_box
+        config.vm.box = vagrant_box_aliases[vagrant_box][0]
+        config.vm.box_version = vagrant_box_aliases[vagrant_box][1]
+      end
```

Comment:
it's interesting that if you set `VAGRANT_BOX=/any/vagrant/box-whatever` we just configure `config.vm.box = ` and move on (see !156) - but if we know that box we do a bunch of tweaking based off some static list we theoretically maintain...

I wonder if we'd better making all these box, box_url, box_version come directly from env (if set) and then maintain some "exmpale" localrc-parallels-template instead?

Reply:
yeah, because maintainers can't test all various vagrant providers I *do* think it'd be easier for "upkeep" to be just READMEs, if you "the m1 guys" all find some new set of vars are working better that what's in the current README then we can just merge that.  If there's enough vbox users we can maintain a board set of "tested" configs I think the README could say that.

I think that "initial bring up" is basically always going to be "clone; try and figure out what provider you should use and follow it's install instructions and edit your localrc like it says; vagrant up"

I don't think it matter so much if you copy & paste one line or 5 lines from the README into your localrc